package.json

```diff
@@ -26,7 +26,7 @@
     "browserify-css": "^0.8.2",
     "debug": "^2.2.0",
     "deep-assign": "^2.0.0",
-    "document-register-element": "^0.5.2",
```

Comment:
I opened a PR on the WebReflection repo. https://github.com/WebReflection/document-register-element/pull/55
Pointing to my personal in the meantime